api/src/test/java/io/grpc/InternalConfigSelectorTest.java

```diff
@@ -55,6 +56,18 @@ public void run() {}
     assertThat(result.getCommittedCallback()).isSameInstanceAs(committedCallback);
   }
 
+  @Test
+  public void resultBuilder_interceptorBased() {
```

Comment:
I don't see any usefulness of such a test, except it increases the code coverage...